testing/tests/DevExpress.ui.widgets.editors/dropDownBox.tests.js

```diff
@@ -493,21 +492,55 @@ QUnit.module('popup options', moduleConfig, () => {
 
         try {
             instance.open();
-
             this.clock.tick();
-            const popup = $('.dx-popup').dxPopup('instance');
+
+            const popup = this.$element.find('.dx-popup').dxPopup('instance');
             const maxHeight = popup.option('maxHeight');
+            assert.roughEqual(Math.floor(maxHeight()), startPopupHeight, 3, 'maxHeight is correct');
```

Comment:
1. do we need so big epsilon?
2. should we use `floor` if we compare values using `roughEqual`?